common/setups/rasr/hybrid_system.py

```diff
@@ -118,7 +118,8 @@ def _add_output_alias_for_train_job(
     ):
         train_job.add_alias(f"train_nn/{train_corpus_key}_{cv_corpus_key}/{name}_train")
         self.jobs[f"{train_corpus_key}_{cv_corpus_key}"][name] = train_job
-        self.nn_models[f"{train_corpus_key}_{cv_corpus_key}"][name] = train_job.out_models
+        if hasattr(train_job, "out_models"):
+            self.nn_models[f"{train_corpus_key}_{cv_corpus_key}"][name] = train_job.out_models
```

Comment:
I am wondering, should we just drop `nn_models` completely? so far everything works via checkpoints...